src/mono/browser/runtime/assets.ts

```diff
@@ -37,7 +37,7 @@ export function instantiate_asset(asset: AssetEntry, url: string, bytes: Uint8Ar
         // falls through
         case "heap":
         case "icu":
-            offset = mono_wasm_load_bytes_into_heap(bytes);
+            offset = mono_wasm_load_bytes_into_heap_persistent(bytes);
```

Comment:
What is the sbrk page size or granularity ?
Are there many assets which could be smaller than the size of the sbrk page ?

Reply:
64k, we could use malloc for small sizes